src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -22,6 +22,7 @@ CA2013 | Reliability | Warning | DoNotUseReferenceEqualsWithValueTypesAnalyzer,
 CA2014 | Reliability | Warning | DoNotUseStackallocInLoopsAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2014)
 CA2015 | Reliability | Warning | DoNotDefineFinalizersForTypesDerivedFromMemoryManager, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2015)
 CA2247 | Usage | Warning | DoNotCreateTaskCompletionSourceWithWrongArguments, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2247)
+CA2248 | Usage | Info | PreferStringContainsOverIndexOf, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca2247)
```

Comment:
Was this file manually changed? If so, please revert this and use the code fix. Also you may want to switch to using CA2249 given https://github.com/dotnet/roslyn-analyzers/pull/3557 and corresponding doc PR https://github.com/MicrosoftDocs/visualstudio-docs/pull/5247/files are already in progress.